qadence_protocols/measurements/protocols.py

```diff
@@ -64,12 +64,12 @@ def verify_robust_shadow(self) -> dict:
         if shadow_groups is None:
             raise KeyError("Shadow protocol requires an option 'shadow_groups' of type 'int'.")
 
-        robust_shadow_correlations = self.options.get("robust_correlations", None)
+        calibration_coefficients = self.options.get("calibration_coefficients", None)
```

Comment:
Maybe simply `calibration` ? Otherwise it makes for a long name.

Reply:
Also, maybe you can default it here if `None` ? Instead of later.

Reply:
It is defaults to `None` with the `get` method.